src/viam/app/app_client.py

```diff
@@ -278,12 +280,58 @@ def proto(self) -> LogEntryPB:
         )
 
 
+class FragmentVisibility(str, Enum):
```

Comment:
In the [proto](https://github.com/viamrobotics/api/blob/c1ef199d69647d3ec7b78c05a7fc44641b644b0f/proto/viam/app/v1/app.proto#L730), this is technically an `int` rather than a string. I don't think it matters, but the string names don't provide any additional data so might be worthwhile changing it to an int to let it match the proto

Reply:
True, but I'm imagining the case where the user is looking at `Fragment.visibility` where it would be more useful to see `"public"` rather than an integer value representing the same thing.